### ISSUE TITLE
Any way to set EXACT window resolution?

### ISSUE THREAD

**Issue by griever-gf** [ata4/angrylion-rdp-plus#81]
This plugin seems just exactly what I want for Densha de Go! 64 translation. Other soft rendering plugins are outdated and slow... (and DX/OGL plugins have gaps between polygons). But I can't set up the EXACT window resolution (I need 1280x960). Yes, I can resize window manually, but it's breaks aspect ratio. And Sizer utility doesn't help with Project 64 window... Any help?

**Comment by ata4** [comment]
Hm... I should implement that in the options, yes. I'll put that on the list.

**Comment by IronYoshi64** [comment]
Yeah I would love to see if you can set the exact window size you want, like 800x600, or 960x720, etc. :)

**Comment by mdtauk** [comment]
Also if there could be a list of pixel multiples based on the original resolution (PAL and NTSC) that would be great...

320 x 240
640 x 480
960 x 720
1280 x 960
1600 x 1200

etc

**Comment by Slider-Whistle** [comment]
@ata4 Waiting keenly. Windowed mode sucks ass when the video window covers the entire screen just to put the game in the bottom left corner. That said, it seems to do exactly the same thing while "Fullscreen" is set to "True". Not entirely sure what's going on there.
It'd be nice if the window were able to rescale itself in these special cases https://www.reddit.com/r/n64/comments/86snpl/which_nintendo_64_titles_change_resolution_while/ but I don't know if many people aside from myself care about that kind of thing.

**Comment by ata4** [comment]
I have added an integer scaling mode for the latest release, which could at least help a bit if I understand the issue correctly. Setting the window size directly is somewhat tricky in Win32 and also wouldn't work in Mupen64Plus, so it's something I'd like to avoid for now.

**Comment by Jj0YzL5nvJ** [comment]
> Setting the window size directly is somewhat tricky in Win32 and also wouldn't work in Mupen64Plus, so it's something I'd like to avoid for now.

`mupen64plus-ui-console.exe --gfx mupen64plus-video-angrylion-plus.dll --rsp mupen64plus-rsp-cxd4.dll --windowed --resolution 1180x885 "Z:\Your_ROM_game.bin"`
https://imgur.com/a/NpXxWh7

It can also be resized in real time...

**Comment by shinslash** [comment]
m64py can set the size of window.
leave the "enable video extension" unchecked.
